animal_owner/views/res_partner.xml

```diff
@@ -0,0 +1,17 @@
+<odoo>
+
+    <!-- Partner Form View -->
+    <record id="view_partner_animal_owner_form" model="ir.ui.view">
+        <field name="name">view.partner.animal.owner.form</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form"/>
+        <field name="arch" type="xml">
+            <notebook position="inside">
```

Comment:
Is it necessary to list animals in a notebook? I sincerely prefer it this way: (but it's a personal option)
https://github.com/odoo/odoo/blob/1527db8cd719c487ace02fca9fde21430e31d350/addons/crm_project/views/crm_lead_views.xml#L10